.github/workflows/test.yml

```diff
@@ -9,15 +9,8 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get install \
-            librust-gstreamer-sys-dev \
-            librust-gtk4-sys-dev \
-            libunwind-dev \
-            librust-gstreamer-sys-dev \
             librust-gdk-sys-dev \
```

Comment:
I believe
```yml
librust-gdk-sys-dev \
```
was missing in the commit that failed on the main branch.

Reply:
https://github.com/xlmnxp/blue-recorder/commit/93a4f02e81b2039bf2475258d05973de8465948e